process/errors.go

```diff
@@ -1145,3 +1145,6 @@ var ErrPropertyTooShort = errors.New("property too short")
 
 // ErrNilProcessDebugger signals that a nil process debugger was provided
 var ErrNilProcessDebugger = errors.New("nil process debugger")
+
+// OutOfAPICalls signals that the allowed max number of calls was reached
+var OutOfAPICalls = errors.New("max api calls reached")
```

Comment:
Consider renaming to `ErrMaxBuiltinCallsReached`.

Reply:
we'll keep it for further suggestions

Reply:
done